google/resource_pubsub_subscription.go

```diff
@@ -186,3 +188,17 @@ func flattenPushConfig(pushConfig *pubsub.PushConfig) []map[string]interface{} {
 
 	return configs
 }
+
+func expandPubsubSubscriptionPushConfig(configured []interface{}) *pubsub.PushConfig {
+	if len(configured) != 1 {
```

Comment:
Minor: `len(configured) == 0` would be more explicit and match the comment

Reply:
Agree, easier to read. Done.